src/UI/Component/Button/Factory.php

```diff
@@ -209,5 +209,55 @@ public function month($default);
 	 */
 	public function tag($label, $action);
 
+	/**
+	 * ---
+	 * description:
+	 *   purpose: >
+	 *     The graphical button combines the recognisability of a graphical element
```

Comment:
The graphical button is highly obtrusive combing a glyph and a textual label on an unusually shaped or sized button. 
The graphical button opens a rich and relevant dropdown. (Page Editor case)
Your case: The graphical button toggles "element"  visible / invisible. (as stated below)

Reply:
I incorporated some of your wording suggestions, but not everything.

Opening a dropdown to me actually is toggling visibilty (at least toggling it from of to on) so it is not that different from opening a slate.